### ISSUE TITLE
Loss function of the father network

### ISSUE THREAD

**Issue by Eric-mingjie** [dhruvramani/Neural-Architecture-Search-with-RL#1]
In the dummy model, network.py, line 30, you define the loss function of the RNN. I find it very weird. Not only does it not follow the original REINFORCE update in the paper, also "tf.reduce_mean(tf.log(1e-10 + hyperparams))" is not understandable to me because from your code, hyperparams is a softmax prediction vector.

**Comment by dhruvramani** [comment]
Yes, the hyperparams is a softmax vector, but the whole thing doesn't end there. Check out the other steps where I multiply the `val_accuracy` with the gradients and apply them. For a clear explanation, check out model_lenet. They both have different code.
